README.md

```diff
@@ -0,0 +1,18 @@
+# rclcpp
+
+This repository contains the source code for the ROS Client Library for C++ package, included with a standard install of any ROS 2 distro.
+
+rclcpp provides the accepted, standard C++ API for interacting with ROS 2.
+
+## Usage
+
+`#include "rclcpp/rclcpp.hpp"` allows use of the most common elements of the ROS 2 system.
+
+Visit the [rclcpp API documentation](http://docs.ros2.org/eloquent/api/rclcpp/) for a complete list of its main components.
+
+### Examples
+
+The ROS 2 tutorials [Writing a simple publisher and subscriber](https://index.ros.org/doc/ros2/Tutorials/Writing-A-Simple-Cpp-Publisher-And-Subscriber/)
+and [Writing a simple service and client](https://index.ros.org/doc/ros2/Tutorials/Writing-A-Simple-Cpp-Service-And-Client/)
+contain examples of rclcpp components in use.
+
```

Comment:
Nit: trailing whitespace.

Reply:
I don't actually see what you're talking about here. I just made this with the GitHub UI, so I don't know if it's a consequence of that, but when I look at the file's changes there's no line 18, but then when I edit it here, there is a line 18.

Reply:
Huh, weird, you are right.  It shows in the Github UI, but not in the file when I check it out.  OK, that's fine, it was a nit anyway :).